peer_mentoring/account_management/models.py

```diff
@@ -31,19 +26,42 @@ class CustomUser(AbstractUser):
     first_name = models.CharField(max_length=150, blank=False, help_text="Required. 150 characters or less.", validators=[MaxLengthValidator(150), MinLengthValidator(1)], error_messages={'blank':'A first name is required.'})
     last_name = models.CharField(max_length=150, blank=False, help_text="Required. 150 characters or less.", validators=[MaxLengthValidator(150), MinLengthValidator(1)], error_messages={'blank':'A last name is required.'})
     dob = models.DateField(blank=False, help_text="Required, please enter a date", )
-    username = models.CharField(max_length=150, unique=True, help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
+    username = models.CharField(max_length=150, blank=False, unique=True, help_text="Required. 150 characters or fewer. Letters, digits and @/./+/-/_ only.",
                                 validators=[username_validator, MaxLengthValidator(150),MinLengthValidator(1)], error_messages={"unique":"A user with that username already exists."})
     email = models.EmailField(unique=True, blank=False, validators=[EmailValidator()] )
     password = models.CharField(blank=False, max_length=127)
     date_joined = models.DateTimeField(default=datetime.now)
 
+    def __str__(self):
+        return self.first_name, self.last_name, self.dob, self.username, self.email, self.date_joined
+
+
+class Photo(models.Model):
+    id = models.BigAutoField(primary_key=True)
+    image = models.ImageField(blank=False, upload_to='photos')
+    description = models.CharField(blank=True, max_length=200)
```

Comment:
some more info on blank vs null: https://stackoverflow.com/questions/8609192/what-is-the-difference-between-null-true-and-blank-true-in-django

Reply:
Went back and evaluated the models and was more purposeful with the chat and text types. Did you want me to review the Django 1.8 response as well? It's cool how that user specified what fields are created in the backend db, but is it relevant to 4.xx or should I look for the docs to get that info on the abstraction?

Reply:
Good point, I did not realize that SO thread spoke about 1.8, although some of this stuff is timeless, better and good reminder to refer to the latest docs for most up2date / accurate infos.

Reply:
Ah, no worries was just wondering if the abstraction changed at all or if it's continued on since 1.8 in how to ORM handles the abstraction for those DBs? Sorry, I haven't been able to look (we got back from the field trip on Sunday afternoon and needed a solid break to re-energize with the family).

Reply:
I am not sure, would have to investigate, but I think the model / ORM stuff is pretty stable / timeless.